sdks/java/core/build.gradle

```
@@ -71,6 +72,7 @@ test {
 
 dependencies {
   antlr library.java.antlr
+  permitUnusedDeclared "org.antlr:antlr4:4.7"
```

permitUnusedDeclared is okay here, but we should add a comment explaining why.

```suggestion
  // antlr is used to generate code from sdks/java/core/src/main/antlr/
  permitUnusedDeclared library.java.antlr
```